utils/shortcodes/picture.js

```diff
@@ -16,7 +16,9 @@ const SIZES = Array.from(new Array(12), (_, index) => (index + 1) * 160)
 const CACHE_FILE = path.join(process.cwd(), '.twelvety.cache')
 
 // Quality of outputted images
-const QUALITY = 75
+const SAMEQUALITY = 75
+const WEBPQUALITY = 60
+const AVIFQUALITY = 35
```

Comment:
Could you add underscores to these variable names?

```suggestion
const SAME_QUALITY = 75
const WEBP_QUALITY = 60
const AVIF_QUALITY = 35
```